Library/Homebrew/formula.rb

```diff
@@ -2096,7 +2096,7 @@ def to_hash
       "version_scheme"           => version_scheme,
       "bottle"                   => {},
       "keg_only"                 => keg_only?,
-      "keg_only_reason"          => keg_only_reason&.to_hash,
+      "keg_only_reason"          => keg_only_reason.gsub(HOMEBREW_PREFIX, HOMEBREW_PREFIX_PLACEHOLDER)&.to_hash,
```

Comment:
Are you 100% sure this is required (it may be, to be clear!). From https://github.com/Homebrew/brew/pull/14615: my thinking was that this substitution would be happen automatically when generating the JSON through `brew generate-formula-api`?

Reply:
If you want to find out: try to run `brew generate-formula-api` without this and see if it's needed.

tbh, though, given https://github.com/Homebrew/brew/pull/15003 this PR can maybe be abandoned?

Reply:
I think this PR can be abandoned since we have the audit now, per https://github.com/Homebrew/brew/pull/15004#issuecomment-1478295975